runtime-decompiler/src/main/java/org/jrd/frontend/PluginMangerFrame/PluginConfigurationEditorController.java

```diff
@@ -74,19 +74,18 @@ public PluginConfigurationEditorController(PluginConfigurationEditorView view, P
             //preparing data for the compiler
             List<URL> dependencyURLs = ((DecompilerWrapperInformation)view.getPluginListPanel().getWrapperJList().getSelectedValue()).getDependencyURLs();
             ArrayList<String> compileStringA = new ArrayList<>();
+            compileStringA.add("-d");
+            compileStringA.add("/tmp");
             compileStringA.add("-cp");
             StringBuilder dependencyS = new StringBuilder();
-            dependencyS.append("\"");
+
             for (URL dependency: dependencyURLs)
             {
                 dependencyS.append(":").append(dependency.getPath());
             }
-            dependencyS.append("\"");
             compileStringA.add(dependencyS.toString());
-            compileStringA.add("-d");
-            compileStringA.add("/tmp");
+
```

Comment:
It is really not obvious what the commit is doing. Do you mind to put more verbose commit message?